compute/compute-node.Dockerfile

```diff
@@ -1264,18 +1264,14 @@ RUN set -e \
     && make -j $(nproc) dist_man_MANS= \
     && make install dist_man_MANS=
 
-#########################################################################################
-#
-# Layers "postgres-exporter", "pgbouncer-exporter", and "sql-exporter"
-#
-#########################################################################################
```

Comment:
And keep this Layer comment block, just check it to `Layer "exporters"` or something. It helps me eprsonally visually separate sections